custom.css

```
@@ -0,0 +1,284 @@
+  border: 1px solid var(--bl-light-border) !important;
+  box-shadow: none !important;
+}
+.dark .code-block {
+  border-color: var(--bl-dark-border) !important;
+}
+/* Shiki paints the panel via `background: var(--shiki-dark-bg)`, applied by
+   Mintlify's `.dark .dark:bg-codeblock` rule with !important. We need both a
+   higher-specificity selector AND to rewrite the var (set inline per block). */
+html.dark .code-block div[class*="bg-codeblock"] {
+  --shiki-dark-bg: var(--bl-dark-code) !important;
+  background-color: var(--bl-dark-code) !important;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)